lib/src/widgets/chat/video/ds_video_message_bubble.widget.dart

```diff
@@ -226,19 +231,33 @@ class _DSVideoMessageBubbleState extends State<DSVideoMessageBubble>
                         )
                       : _buidErrorIcon(),
             ),
-            if (widget.text?.isNotEmpty ?? false)
+            if ((widget.title?.isNotEmpty ?? false) ||
```

Comment:
Since this test is done multiple times, you could create a variable that holds this value and use it along the way.

`final hasText = widget.text?.isNotEmpty ?? false`
`final hasTitle = widget.title?.isNotEmpty ?? false`

Reply:
Done!